packages/react-server-dom-webpack/src/ReactFlightWebpackPlugin.js

```diff
@@ -248,25 +246,35 @@ export default class ReactFlightWebpackPlugin {
                 .getExportsInfo(module)
                 .getProvidedExports();
 
-              const clientExports: {
-                [string]: {chunks: $FlowFixMe, id: $FlowFixMe, name: string},
-              } = {};
-
-              const ssrExports: {
-                [string]: {specifier: string, name: string},
-              } = {};
-
               const href = pathToFileURL(module.resource).href;
 
               if (href !== undefined) {
-                ['', '*']
-                  .concat(
-                    Array.isArray(moduleProvidedExports)
-                      ? moduleProvidedExports
-                      : [],
-                  )
-                  .forEach(function (name) {
-                    clientExports[name] = {
+                const ssrExports: {
+                  [string]: {specifier: string, name: string},
+                } = {};
+
+                clientManifest[href] = {
+                  id,
+                  chunks: chunkIds,
+                  name: '*',
+                };
```

Comment:
@shuding Note that importing all exports ('*') is encoded using only the href.

Importing defaults conditionally is encoding using `#`.

Named exports are encoded using `'#' + name`.

The bundler need to update accordingly. It doesn't really matter what it is along as the loaders do the same thing.